book/10-git-internals/sections/objects.asc

```diff
@@ -139,6 +139,15 @@ $ git cat-file -p master^{tree}
 The `master^{tree}` syntax specifies the tree object that is pointed to by the last commit on your `master` branch.
 Notice that the `lib` subdirectory isn't a blob but a pointer to another tree:
 
+[NOTE]
+====
+Depending on what shell you use, you may encounter errors when using the `master^{tree}` syntax.
+
+In PowerShell or CMD on Windows, the `^` character is used for escaping, so you have to double it to avoid this: `git cat-file -p mater^^{tree}`.
+
+If you're using ZSH, the `^` character is used for globbing, so you have to enclose the whole expression in quotes: `git cat-file -p "master^{tree}"`
+====
+
```

Comment:
Oh man, I hate to do this to you again, but I have one last request (I promise). Looking at how this renders in context:

![image](https://user-images.githubusercontent.com/39902/37368253-e9321a4c-26c2-11e8-9cd5-f6aca00c3c54.png)

The "here's what you should do, and here's what it looks like when you do it" flow is interrupted by this note block. Could you move it just below the code block?

Reply:
Sure no problem

Reply:
Hope these formatting rules are defined somewhere so that people could just read and follow it.